test/unit/bin/ember-template-lint-test.js

```diff
@@ -89,6 +89,93 @@ describe('ember-template-lint executable', function() {
       });
     });
 
+    describe('given no path', function() {
+      it('should print errors', function(done) {
+        execFile(
+          'node',
+          ['../../../bin/ember-template-lint.js', '<', 'app/templates/application.hbs'],
+          {
+            cwd: './test/fixtures/with-errors',
+            shell: true,
+          },
+          function(err, stdout, stderr) {
+            expect(err).to.be.ok;
+            expect(stdout).to.be.ok;
+            expect(stderr).to.be.empty;
+            done();
+          }
+        );
+      });
+    });
+
+    describe('given no path with --filename', function() {
+      it('should print errors', function(done) {
+        execFile(
+          'node',
+          [
+            '../../../bin/ember-template-lint.js',
+            '--filename',
+            'app/templates/application.hbs',
+            '<',
+            'app/templates/application.hbs',
+          ],
+          {
+            cwd: './test/fixtures/with-errors',
```

Comment:
I think we want these "implicit stdin" tests to be done in a directory that is **not** the same as the tests above. We want to confirm that the linter is properly linting the stdin and _cannot_ be reading the file from the `--filename`.